airflow/models/dag.py

```diff
@@ -2495,11 +2495,15 @@ def bulk_write_to_db(cls, dags: Collection["DAG"], session=NEW_SESSION):
             orm_dag.last_parsed_time = timezone.utcnow()
             orm_dag.default_view = dag.default_view
             orm_dag.description = dag.description
-            orm_dag.schedule_interval = dag.schedule_interval
             orm_dag.max_active_tasks = dag.max_active_tasks
             orm_dag.max_active_runs = dag.max_active_runs
             orm_dag.has_task_concurrency_limits = any(t.max_active_tis_per_dag is not None for t in dag.tasks)
-            orm_dag.timetable_description = dag.timetable.description
+            if dag.schedule_on:
+                orm_dag.schedule_interval = 'dataset-triggered'
```

Comment:
```suggestion
                orm_dag.schedule_interval = 'Dataset Triggered'
```

Should we make this more human oriented?

Reply:
If this is just a label to display, I'd say so. But if we plan to use it anywhere else maybe it should stay as-is/

Reply:
After AIP-39 we now only use `dag.schedule_interval` for display. (I wonder if it’d be cleaner to create a custom timetable for dataset-triggered DAGs, this would make this part easier to follow and have less special cases.)

Reply:
Yeah. Sounds like a good idea. And maybe worth considering renaming to "schedule short name" and "schedule long name" or rather than interval and description? (timetables don't necessarily have an interval)

Reply:
I think "description" is fine. The other could be "label"

Reply:
i'm talking about the dag orm --- are you suggesting `schedule_description` and `schedule_label`?

Reply:
_Description_ is already used (it’s the thing in the tooltip popup). Perhaps `timetable_summary` since that’s where the value comes from?